nixos/modules/services/logging/logrotate.nix

```diff
@@ -16,8 +16,13 @@ let
         '';
       };
 
-      path = mkOption {
+      name = mkOption {
         type = types.str;
+        internal = true;
+      };
+
+      path = mkOption {
```

Comment:
> Note that the double quotes around the first filename at the beginning of this section allows logrotate to rotate logs with spaces in the name. Normal shell quoting rules apply, with ', ", and \ characters supported. 

It might be worth mentioning that we are quoting entries now and what that implies (that the user should not).